package.json

```diff
@@ -49,7 +49,8 @@
     "prettier": "~3.1.0",
     "rimraf": "^5.0.5",
     "serve": "^14.2.1",
-    "turbo": "~1.12.3"
+    "turbo": "~1.12.3",
+    "configs": "workspace:*"
```

Comment:
Does this mean you can remove the individual `"configs"` from deps in all workspaces?

Reply:
Yes, I believe so. `install`, `dev`, and `build` still work.

I also made dummy changes in all workspaces. When `configs` was not declared as a dep in any workspace, I got this error:

<img width="1073" alt="image" src="https://github.com/iTwin/iTwinUI/assets/45748283/b2724b03-a147-499f-9f70-39ae43803ecf">

But after adding `configs` to the devDeps of only the root package.json, I was able to make the commit.

So removed the `configs` devDep from other workspaces (https://github.com/iTwin/iTwinUI/pull/1874/commits/3dd4144af7d9e051fdfaf268249cebb8c9ffdd7b).